simexpal/launch/common.py

```diff
@@ -26,7 +26,7 @@ def lock_run(run):
 	# can be considered a bug (or deliberate misuse) and will lead to hard failures.
 	try:
 		lockfd = os.open(run.aux_file_path('lock'),
-				os.O_RDONLY | os.O_CREAT | os.O_EXCL, mode=0)
+				os.O_RDONLY | os.O_CREAT | os.O_EXCL, mode=0o444)
```

Comment:
The mode should be set to read and write (should be `0o666` instead of `0o444`).

Reply:
actually 444 is fine for copying as well, we can keep that.